deploy/docker/dev.sh

```diff
@@ -3,4 +3,6 @@
 /bin/sh /code/deploy/docker/common.sh
 /code/deploy/docker/wait-for-it.sh db:5432 -- invoke restore_from_dump
 /code/deploy/docker/wait-for-it.sh solr:8983 -- python3 manage.py rebuild_index --noinput
+/code/deploy/docker/wait-for-it.sh redis:6379 -- redis-server
```

Comment:
Not sure what this line is supposed to be doing but it seems wrong. The redis server should be started already in the Docker redis container. Will remove.